### DISCUSSION TITLE
Is ResetSceneItem still a thing?

### DISCUSSION THREAD

**Discussion by London-Lass** [Palakis/obs-websocket#1158] [General]
Currently trying to convert from 4.9 to 5.0. I can't find any mention of ResetSceneItem in Protocol.md. Is there an alternative?

**Comment by tt2468** [comment]
Send a `SetInputSettings` with `{}` as the `inputSettings` field, and it will do the same thing.

**Comment by London-Lass** [comment]
That did the job. Thank you.
